### PR TITLE
edited the main handlebars. Just need to finish the body rendering here.

### PR DESCRIPTION
**Pull request by ilyublinsky**

### CODE REVIEW

File: views/htmls/main.html [hunk 1]
```diff
@@ -2,67 +2,60 @@
...
+    {{#if logged_in}}
+    <script src="/js/logout.js"></script>
+    {{/if}}
+
     <footer>
-      
+        <nav class="navbar navbar-expand-lg fixed-bottom navbar-color bg-color">
+            <a class="navbar-brand" href="#">
+                <li class="nav-item active">
+                    <a href="#placeholder" class="fa fa-facebook"></a>
```
Comment (MikeyP957) [inline views/htmls/main.html:49]: missing <ul>

File: views/htmls/main.html [hunk 2]
```diff
@@ -2,67 +2,60 @@
...
+    <!-- Render script for logged in users only -->
+    {{#if logged_in}}
+    <script src="/js/logout.js"></script>
+    {{/if}}
+
     <footer>
-      
+        <nav class="navbar navbar-expand-lg fixed-bottom navbar-color bg-color">
+            <a class="navbar-brand" href="#">
+                <li class="nav-item active">
```
Comment (MikeyP957) [inline views/htmls/main.html:48]: Should this be <ul> ?
I see </ul> but no open tag.

Comment (MikeyP957) [inline views/htmls/main.html:48]: should this be '<ul>' 

I see a '</ul>' but no open tag

Comment (MikeyP957) [inline views/htmls/main.html:48]: missing open tag for ul

**Review by MikeyP957 — CHANGES_REQUESTED** [review]
I just see one typo

**Comment by ilyublinsky** [comment]
Thanks!

**Pull request closed without merging**